openkfw/TruBudget
# 1870/1886/1884: Change wordings

### Checklist




### Description
- change of wordings for issues #1870 #1886 and #1884

### How to test

1. <E.g. login as user XYZ>
2. <E.g. try to create new subproject>
3. <E.g. You should not see error in console>
4. ...

Closes #1870 #1886 #1884
